var/spack/repos/builtin/packages/miopen-hip/package.py

```diff
@@ -84,12 +85,31 @@ class MiopenHip(CMakePackage):
         "6.1.0",
         "6.1.1",
         "6.1.2",
+        "6.2.0",
     ]:
         depends_on(f"rocm-cmake@{ver}:", type="build", when=f"@{ver}")
         depends_on(f"hip@{ver}", when=f"@{ver}")
-        depends_on(f"rocm-clang-ocl@{ver}", when=f"@{ver}")
         depends_on(f"rocblas@{ver}", when=f"@{ver}")
 
+    for ver in [
+        "5.3.0",
+        "5.3.3",
+        "5.4.0",
+        "5.4.3",
+        "5.5.0",
+        "5.5.1",
+        "5.6.0",
+        "5.6.1",
+        "5.7.0",
+        "5.7.1",
+        "6.0.0",
+        "6.0.2",
+        "6.1.0",
+        "6.1.1",
+        "6.1.2",
+    ]:
+        depends_on(f"rocm-clang-ocl@{ver}", when=f"@{ver}")
+
```

Comment:
Does this change mean this package no longer depends on `rocm-clang-ocl` for its versions `@6.2.0:`?

Reply:
Yes, this package will no longer depend on rocm-clang-ocl from 6.2 onwards. Rocm-clang-ocl is deprecated now, 6.1.2 was the final release.

Reply:
Thanks for the confirmation.